example-mojaloop-backend/values.yaml

```diff
@@ -3,8 +3,15 @@
 # Declare global configurations
 global: {}
 
+nginx:
+  enabled: true
```

Comment:
Please provide a "reference" so there is a link to easily see the values etc.

Reply:
@mdebarros : Good Point !

Reply:
@mdebarros : actually on reflection this does not need to be here per our earlier agreement conversation nginx is not part of the backend because it is often part of the product and having it in the backend is going to be unreliable at best and unreliable and confusing at worst.